sandbox/registration-processor-mz.properties

```diff
@@ -693,8 +690,8 @@ mosip.regproc.packet.classifier.tagging.exceptionbiometrics.tag-name=EXCEPTION_B
 # This mapping will contain the short words for each missing biometrics, the values will used for concatenating in the tags
 mosip.regproc.packet.classifier.tagging.exceptionbiometrics.bio-value-mapping={'leftLittle':'LL','leftRing':'LR','leftMiddle':'LM','leftIndex':'LI','leftThumb':'LT','rightLittle':'RL','rightRing':'RR','rightMiddle':'RM','rightIndex':'RI','rightThumb':'RT','leftEye':'LE','rightEye':'RE'}
 
-mosip.regproc.camelbridge.intercept-hotlisted-key=HOTLISTED
-mosip.regproc.camelbridge.pause-settings=[{"hotlistedReason": "operator","pauseFor": 432000,"defaultResumeAction": "RESUME_PROCESSING_AND_REMOVE_HOTLISTED_TAG","fromAddress": ".*"},{"hotlistedReason": "center","pauseFor": 600,"defaultResumeAction": "STOP_PROCESSING","fromAddress": "eventbus://packet-validator-new-bus-out"}]
+
+mosip.regproc.camelbridge.pause-settings=[{"matchExpression": "$.tags[?(@['META_INFO-META_DATA-centerId'] == '10002')]","pauseFor": 432000,"defaultResumeAction": "RESUME_PROCESSING","fromAddress": ".*"},{"matchExpression": "$.tags[?(@['META_INFO-META_DATA-machineId'] == '10074')]","pauseFor": 600,"defaultResumeAction": "STOP_PROCESSING","fromAddress": "eventbus://packet-validator-new-bus-out"}]
```

Comment:
use the match expression with HOTLISTED = operator and other condition as before

Reply:
Also the resume remove tags is missing